keylime/migrations/versions/4329e2d14944_associate_moved_allowlists_to_agents.py

```diff
@@ -0,0 +1,55 @@
+"""Associate moved allowlists to agents (2/2)
```

Comment:
Is there a reason you are using 2 migrations here? Typically a single change will use a single migration just so everything is done at once.

Reply:
I was running into issues attempting to populate columns (specifically the new foreign key column) that hadn't been committed yet. Adding a second migration to populate the new columns after they'd been committed by the first migration seems to work, so that's what I did. And vice versa for the downgrade step.